docs/computing/disk.md

```diff
@@ -126,3 +126,20 @@ quota regardless of how much data you actually have in the scratch
 directory. See [billing](../accounts/billing.md) for details.
 Furthermore, even after the quota is increased, the automatic cleaning
 process will continue removing idle files from the _scratch_ directory.
+
+
+## Additional disk areas
+
+### Login nodes
+
+All of the login nodes have 2900 GiB of fast local storage. The storage
+is located under `$TMPDIR` and is separate for each login node.  
+
+The local storage is good for compiling applications and performing 
+pre- and postprocessing that require heavy IO operations, for example packing and unpacking 
+archive files. Remember to move your data to a shared disk area afterwards. 
```

Comment:
is there any cleaning of this data?

Reply:
Clean up if files have not been accessed in 10 days
`/usr/sbin/tmpwatch "$flags" -x /local_scratch/build 10d /local_scratch`
Should we mention it explicitly or just say that files are frequently cleaned?